addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/state/ZoneControlState.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.state;
+
+/**
+ * The state of a specific zone of a Yamaha receiver.
+ *
+ * @author David Graeff <david.graeff@web.de>
+ */
+public class ZoneControlState {
+    public boolean power = false;
```

Comment:
Yup, if a receiver does not support all fields of a state, the default one is used, which usually represents an Off/Not available/Invalid state.